crates/bevy_ui/src/render/mod.rs

```diff
@@ -58,8 +58,10 @@ pub const UI_SHADER_HANDLE: Handle<Shader> = Handle::weak_from_u128(130128470471
 
 #[derive(Debug, Hash, PartialEq, Eq, Clone, SystemSet)]
 pub enum RenderUiSystem {
-    ExtractNode,
-    ExtractAtlasNode,
+    ExtractBackground,
+    ExtractDecoration,
+    ExtractImage,
+    ExtractText,
```

Comment:
The systems extract multiple items, so the names should be plural.

```suggestion
    ExtractBackgroundColors,
    ExtractImages,
    ExtractBorders,
    ExtractText,
```

Also rearranged to be in what should be the order of execution.

Reply:
Rearranged. Assuming the rename from `ExtractDecorations` to `ExtractBorders` is not meant to be part of this suggestion?

Reply:
I missed the explanation for the rename to `ExtractBorders` in my previous reply. I don't feel strongly about this but I do prefer `ExtractDecorations`, because the name `ExtractBorders` is ambiguous (is it referring to _borders_ borders, or the set of border types that also includes outlines?). And in the future there may be other decorations.

Reply:
Going to apply this suggestion anyways because I don't feel strongly.